modular_zubbers/code/modules/clothing_reskins/redsec.dm

```diff
@@ -0,0 +1,65 @@
+//Adds Red Security outfit variants to the security equipment reskin menu.
+
+
+/obj/item/storage/backpack/security/Initialize(mapload)
+	unique_reskin["RedSec Variant"] = list(
+			RESKIN_ICON = 'icons/obj/storage/backpack.dmi',
+			RESKIN_ICON_STATE = "backpack-security",
+			RESKIN_WORN_ICON = 'icons/mob/clothing/back/backpack.dmi',
+			RESKIN_INHAND_L = 'icons/mob/inhands/equipment/backpack_lefthand.dmi',
+			RESKIN_INHAND_R = 'icons/mob/inhands/equipment/backpack_righthand.dmi',
+			RESKIN_INHAND_STATE = "securitypack"
+		)
+	. = ..()
+
+/obj/item/storage/backpack/satchel/sec/Initialize(mapload)
+	unique_reskin["RedSec Variant"] = list(
+			RESKIN_ICON = 'icons/obj/storage/backpack.dmi',
+			RESKIN_ICON_STATE = "satchel-security",
+			RESKIN_WORN_ICON = 'icons/mob/clothing/back/backpack.dmi',
+			RESKIN_INHAND_L = 'icons/mob/inhands/equipment/backpack_lefthand.dmi',
+			RESKIN_INHAND_R = 'icons/mob/inhands/equipment/backpack_righthand.dmi',
+			RESKIN_INHAND_STATE = "satchel-sec"
+		)
+	. = ..()
+
+/obj/item/storage/backpack/duffelbag/sec/Initialize(mapload)
+	unique_reskin["RedSec Variant"] = list(
+			RESKIN_ICON = 'icons/obj/storage/backpack.dmi',
+			RESKIN_ICON_STATE = "duffel-security",
+			RESKIN_WORN_ICON = 'icons/mob/clothing/back/backpack.dmi',
+			RESKIN_INHAND_L = 'icons/mob/inhands/equipment/backpack_lefthand.dmi',
+			RESKIN_INHAND_R = 'icons/mob/inhands/equipment/backpack_righthand.dmi',
+			RESKIN_INHAND_STATE = "duffel-sec"
+		)
+	. = ..()
+
+/obj/item/clothing/head/security_cap/Initialize(mapload)
+	unique_reskin["RedSec Variant"] = list(
+			RESKIN_ICON = 'icons/obj/clothing/head/hats.dmi',
+			RESKIN_ICON_STATE = "secsoft",
+			RESKIN_WORN_ICON = 'icons/mob/clothing/head/hats.dmi'
+		)
+	. = ..()
+
+/obj/item/clothing/gloves/color/black/security/Initialize(mapload)
+	unique_reskin["RedSec Variant"] = list(
+			RESKIN_ICON = 'icons/obj/clothing/gloves.dmi',
+			RESKIN_ICON_STATE = "black",
+			RESKIN_WORN_ICON = 'icons/mob/clothing/hands.dmi'
+		)
+	. = ..()
+
+/obj/item/clothing/under/rank/security/officer/Initialize(mapload)
+	unique_reskin["RedSec Variant"] = list(
+			RESKIN_ICON_STATE = "rsecurity",
+			RESKIN_WORN_ICON_STATE = "rsecurity"
+		)
+	. = ..()
```

Comment:
Ok, problem here: We can't add reskinning on this specific piece of clothing, because altclick already handles changing how you wear it.
Also this might break subtypes like /obj/item/clothing/under/rank/security/officer/grey

Reply:
All the items listed, including _that_ one already have reskinning on them. I'm just adding an additional option to them(Red Sec) You're right, it does cause you to set the outfit's adjustment to casual while also changing the look of it, but you can just alt click it again to adjust it back since you can only reskin once per clothing item. So it does work even though you'd effectively have to alt-click it twice(Reskin + Adjust, Adjust back). 
I really do wish the original Reskin code was not on Alt+click though. Part of the reason why I didn't add Winter coats to this list is because it's got a different alt+click adjustment code that makes it impossible to use with the reskin code as is.

Reply:
Alright. After testing some things out.
So that uniform is already reskinnable, so I am not adding that particular functionality. As for the subtypes, they are already setup with flags to prevent from "inheriting" that reskin feature(Except for gray, which is just a gray variant that's unobtainable? but seems to offer the reskin option currently, likely intentionally). So the orderly for example can't reskin his outfit. 
However doing it in that particular modular method definitely breaks a lot of the subtypes.
So I think I might have to go semi-modular for the sake of the uniform since it's so high up the hierarchy. This allows it to function and keeps all the subtypes working appropriately.